package.json

```diff
@@ -10,7 +10,8 @@
     "react": "16.0.0",
     "react-native": "https://github.com/expo/react-native/archive/sdk-24.0.0.tar.gz",
     "react-native-collapsible": "~0.10.0",
-    "react-native-elements": "~0.19.0",
+    "react-native-elements": "~1.0.0-beta4",
+    "react-native-keyboard-aware-scroll-view": "^0.5.0",
```

Comment:
Going to try this library out to see if it makes things smoother. It scrolls to the focused field when we open a keyboard. If it works, it should be handy for the `AddSiteFormScreen`